imap_processing/glows/l0/decom_glows.py

```diff
@@ -0,0 +1,77 @@
+from enum import Enum
+from pathlib import Path
+
+from bitstring import ReadError
+from space_packet_parser import parser, xtcedef
+
+from imap_processing import imap_module_directory
+from imap_processing.glows.l0.glows_l0_data import DirectEventL0, HistogramL0
+
+
+class GlowsParams(Enum):
```

Comment:
```suggestion
class GlowsParams(IntEnum):
```
so then below you can do `if x == GlowsParams.HIST_APID` instead of having to get into the `.value`